docs/application/dotnet/guides/nui/styling-controls-with-xaml.md

```diff
@@ -0,0 +1,201 @@
+# Styling UI components with XAML
+Styling of components in NUI is possible in two ways:
+  - `ViewStyle`: designed for simple themes.
+  -  XAML file: designed for advanced themes. With XAML file, styles change in runtime and can be shared between the applications.
+
+## ViewStyle theme
+The basic styling of NUI components is based on the `ViewStyle` class and its derivatives. `ViewStyle` defines attributes and stores their values. 
+
+The following is a list of the most common attributes:  
+- name
+- state 
+- subState
+- position
+- scale
+- orientation
+- padding 
+
+You can find all the attributes and their details on [ViewStyle attributes](https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/public/BaseComponents/Style/ViewStyle.cs) page.
+
+In addition, styles can also be defined for the components with following states:
```

Comment:
@mihashco 
Lines 20-30

In addition, styles can also be defined for the components with the following states:

| State              | Description                                                                                                                                          |
|--------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|
| `Normal`           | The default style.                                                                                                                         |
| `Focused`          | The state where all the keystrokes and events are received by the widget. There can only be one widget in this state at a time in the active window. |
| `Disabled`         | The state where the widget is disabled. All other events are ignored in this state.                                                          |
| `Selected`         | The state where the widget is selected.                                                                                                      |
| `Pressed`          | The state where the widget receives touch event.                                                                                                                                                   |
| `DisabledFocused`  | The state that is similar to the `Focused` state, but there is no possibility to change the widget state.                                                                        |
| `SelectedFocused`  | The state where the widget is checked and all events are received.                                                                                            |
| `DisabledSelected` | The combination of disabled and selected states.                                                                                                       |